content/en/_index.html

```diff
@@ -3,26 +3,138 @@
+</p>
+<p>
+	The secondary use case is observability and metrics. Essential to improving the Continuous Delivery workflow is the ability of the pipeline to collect events from different CI/CD tools. Common events, reported from all involved tools, permits a unified, end-to-end process that spans many tools.
+</p>
+{{< /blocks/lead >}}
+
+{{< blocks/lead title="events" color="light">}}
+<p>
+	The goal of the CDEvents specification is to define interoperability between systems that allow services to produce or consume events, where the producer and consumer can be developed and deployed independently, in isolation. A producer can generate events before a consumer is listening, and a consumer can express an interest in an event or class of events that is not yet being produced.
+</p>
+{{< /blocks/lead >}}
+
+{{< blocks/lead title="Benefits" color="white">}}
+<p>
+	The CDEvents specification extends the <a href="https://cloudevents.io/">CloudEvents</a> specification, providing a vocabulary appropriate to CI/CD events. This vocabulary is built upon the <a href="https://bestpractices.cd.foundation/resources/tools-terminology/">terminology</a> created and maintained within the CDF interoperability special interest group.
+</p>
+<p>
+	The intention is to provide an Open framework in which you can describe your desired Continuous Delivery processes, in line with best practices and expressed in a familiar form that is aligned to your other Cloud-based event-driven processes.
+</p>
+<p>
+	CDEvents introduces a set of common abstractions to aid in this process. 
+</p>
 
-CDEvents is a common specification for Continuous Delivery events, enabling interoperability in the complete software production ecosystem.
 <br/>
-It's an incubated project at the <a href="https://cd.foundation/projects">Continuous Delivery Foundation</a> (CDF).
+
+{{< cardpane >}}
+	{{< card header="Source Code Version Control Events" >}}
+	Represent events associated with the management of source code assets
+	{{< /card >}}
+	{{< card header="Continuous Integration Pipelines Events" >}}
+	Represent events related to building, testing, packaging, and releasing software artifacts
+	{{< /card >}}
+{{< /cardpane >}}
+{{< cardpane >}}
+	{{< card header="Continuous Deployment Pipelines Events" >}}
+	Represent events related to environments where the artifacts produced by the integration pipelines are intended to run
+	{{< /card >}}
+	{{< card header="Core Events" >}}
+	Represent shared abstractions such as Task Runs and Pipeline Runs
+	{{< /card >}}
+{{< /cardpane >}}
+
+<p>
+	You can find CDEvents support being implemented in many of your favorite CI/CD tools, including Argo, Jenkins, Jenkins-X, Keptn, Knative, Ortelius, Tekton
```

Comment:
I guess the order of tools here is alphabetical, right?
This is a nice sentence to have, perhaps "being implemented" is not close enough to reality for some of these tools at least. 
- ArgoCD: If I'm not mistaken @bradmccoydev has done a demo with ArgoCD and CDEvents, but I'm not sure if we've actually worked with the Argo community yet. 
- Jenkins-X has expressed interest in CDEvents
- I'm not aware of any Ortelius work with CDEvents, but I may have missed it
- Spinnaker is very interested in CDEvents, so we should include it in the list
- Shipwright as expressed interest too
- @bradmccoydev has worked with Flux (and the Flux community) on a POC with Flux

Reply:
Happy for you to define what the list should be, today.

There was an Ortelius presentation at CDEventsCon and there was at least one team member in this week's meeting.